src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -39,7 +43,7 @@ const StyledLogo = styled.img`
   margin-left: 6px;
 `
 
-export function ConfirmationPendingContent({
+function ConfirmationPendingContent({
```

Comment:
@biocom @W3stside , not sure if this is what I think it is, but it looks like something we want and Michel was going to work in. Right now there/s no modal confirmation when you approve/wrap/unwrap, so we would need one. 

Is this it?